proto/viam/service/slam/v1/slam.proto

```diff
@@ -26,6 +26,29 @@ service SLAMService {
       get: "/viam/api/v1/service/slam/{name}/map"
     };
   }
+
+  // GetPositionNew returns the current estimated position of the robot with
```

Comment:
Using the word New in GetPositionNew confuses me as to what it would return to me until after reading the comment. What about GetEstimatedPosition?

Reply:
We won't have GetPositionNew eventually. This is to allow us to do all the development and then swap the functions that get called in the UI. See tech spec: https://docs.google.com/document/d/1GAnAGrv0gcQkIMzY8x3NNanigRzkm6oPH6pKTHg8XjY/edit

Reply:
Maybe a comment explaining that then would help. Otherwise it makes New seem like it's creating something to an unaware user.

Reply:
Adding a comment to note that this function will be replacing the current `GetPosition`